### ISSUE TITLE
.xcworkspace does not exist.

### ISSUE THREAD

**Issue by MillerApps** [asmallteapot/cocoapods-playgrounds#43]
When running pod playgrounds all the files are created but they claim the .xcworkspace does not exist, even though it does. When opening the playground the error no such module is shown. I am using version 1.2.2 and Cocoapods-1.1.0.rc.2.

**Comment by juggernate** [comment]
Same here with 1.2.2, cocoapods 1.0.1. Sierra and Xcode 8

**Comment by MillerApps** [comment]
For the record I am also running Sierra and Xcode 8 (both are app store versions).

**Comment by edasque** [comment]
Same here. El Capitan, Xcode 8, cocoapods 1.1.0.rc.2

**Comment by segiddins** [comment]
Can someone please share concrete steps to reproduce the issue? Thanks!

**Comment by edasque** [comment]
``` bash
➜  iOS_Dev mkdir LambdaPG
➜  iOS_Dev cd LambdaPG
➜  LambdaPG pod playgrounds AWSLambda
The file /Users/edasque/Dropbox/iOS_Dev/LambdaPG/AWSLambdaPlayground/AWSLambdaPlayground/AWSLambda.xcworkspace does not exist.
```

**Comment by segiddins** [comment]
Thanks!

**Comment by edasque** [comment]
I am guessing it's an Xcode 8 issue.

**Comment by aofeng2009** [comment]
Same here.  Using version Xcode8 ,cocoapods 1.0.1

**Comment by edasque** [comment]
Any update, @segiddins ?

**Comment by RafaelPlantard** [comment]
Same here!!! Any news @segiddins ???

**Comment by segiddins** [comment]
I've been busy working on other things the past few weeks, but if anyone wants to make a PR improving the situation, I'd be happy to review it 👯‍♂️

**Comment by icanzilb** [comment]
@segiddins has this project been abandoned? there is a PR fixing this issue that is ~ 2 months old

**Comment by edasque** [comment]
What is the way to install @askielboe's fork ? I know I did so in the past to test that it worked but I can't remember how.

**Comment by segiddins** [comment]
@icanzilb my comments on that PR still haven't been addressed

**Comment by asmallteapot** [comment]
Fixed in #56. A new release is coming soon!
